recipes/font-ttf-noto/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/fonts || true
+install -v -m644 unhinted/variable-ttf/*.ttf ${PREFIX}/fonts/
```

Comment:
So the actual conda-forge package size is much smaller than 1GB upstream source release because you are only including TTF files from this single directory? Can you comment why you have chosen the unhinted variable TTF fonts instead of the OTF fonts or the hinted TTF fonts?

Reply:
Yes, I think we can get the size down. I chose .ttf because that's what other font package in conda-forge seem to provide. I chose unhinted because this is the only format that provides variable fonts as far as I can tell.